pkg/grizzly/workflow.go

```diff
@@ -103,14 +103,20 @@ func ListRemote(opts Opts) error {
 	return w.Flush()
 }
 
-// Pulls remote resources
+// Pull pulls remote resources and stores them in the local file system.
+// The given resourcePath must be a directory, where all resources will be stored.
 func Pull(resourcePath string, opts Opts) error {
-	log.Infof("Pulling resources from %s", resourcePath)
+	stat, err := os.Stat(resourcePath)
+	if err != nil {
```

Comment:
It's probably good idea to omit `ErrNotExist` errors here, cause in _Pull_ operations, the local directory might not exist, and can _just_ be created. Right? Wdyt? @malcolmholmes

Reply:
pull requires a dir, no? In which case, sure it can make the dir.

Reply:
Yeah, agreed! I added a bunch of changes to handle that, as well as to test the expected behavior!